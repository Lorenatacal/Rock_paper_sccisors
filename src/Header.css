.Header {
  margin-bottom: 3%;
}
.App-header {
  background-color: white;
  margin-top: 1%;
  align-items: center;
  font-size: 90%;
  color: black;
}

.App-logo {
  margin: -2%;
  animation: App-logo-spin infinite 40s linear;
  height: 20vmin;
  pointer-events: none;
}

@keyframes App-logo-spin {
  from {
    transform: rotate(0deg);
  }
  to {
    transform: rotate(360deg);
  }
}
