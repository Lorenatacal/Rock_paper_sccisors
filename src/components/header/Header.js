import React, { Component } from "react";
import styled from "styled-components";

const StyledHeader = styled.div`
  margin-bottom: 2%;
`;
const StyledAppHeader = styled.header`
  margin-left: 1%;
  margin-right: 1%;
  background-color: rgb(241, 239, 239);
  font-size: 90%;
  color: black;
`;
const StyledTitle = styled.h2`
  margin-left: 20%;
  margin-top: 1%;
  position: absolute;
`;
const StyledLogo = styled.img`
  animation: App-logo-spin infinite 40s linear;
  height: 13vmin;
  @keyframes App-logo-spin {
    from {
      transform: rotate(0deg);
    }
    to {
      transform: rotate(360deg);
    }
  }
  margin-left: 65%;
  margin-top: 1%;
`;

class Header extends Component {
  render() {
    return (
      <StyledHeader>
        <StyledAppHeader>
          <StyledTitle>Play Rock Paper Scissors</StyledTitle>
          <StyledLogo
            src="http://www.sclance.com/pngs/rock-paper-scissors-png/rock_paper_scissors_png_1176896.png"
            alt="logo"
            name="RockPaperScissors"
          />
        </StyledAppHeader>
      </StyledHeader>
    );
  }
}

export default Header;
