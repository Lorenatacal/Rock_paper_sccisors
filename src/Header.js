import React, { Component } from 'react';
import './Header.css';

class Header extends Component {
  render() {
    return (
      <div className="App">
        <header className="App-header">
          <h2>Play Rock Paper Scissors</h2>
          <img src="https://png2.kisspng.com/sh/1cee9ac5ea79840080abb5155caeacf0/L0KzQYm3VsI6N5R3eZH0aYP2gLBuTgJwa5x1eeJucoPmecT6jCJ0NaR5fdd2aYSwfLFuj71zd5RwReJqcHX1PcTqigN0d6N4RadrM0a0c4eCgBE5O2E2RqcEMUG3QoK9UcU0OGM8SqoCM0i1SYa1kP5o/kisspng-rockpaperscissors-steemit-logo-rock-paper-scissors-5b361c69ca8301.5911421615302728738295.png" className="App-logo" alt="logo" />
        </header>
      </div>
    );
  }
}

export default Header;