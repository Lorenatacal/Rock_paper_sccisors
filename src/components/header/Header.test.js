import React, { Component } from "react";
import Header from "./Header";
import Enzyme from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import toJson from "enzyme-to-json";

Enzyme.configure({ adapter: new Adapter() });

test("Header should render correctly", () => {
  const wrapper = Enzyme.shallow(<Header />);
  expect(toJson(wrapper)).toMatchSnapshot();
});
