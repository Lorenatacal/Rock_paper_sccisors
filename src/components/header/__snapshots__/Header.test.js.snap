// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render correctly 1`] = `
<styled.div>
  <styled.header>
    <styled.h2>
      Play Rock Paper Scissors
    </styled.h2>
    <styled.img
      alt="logo"
      name="RockPaperScissors"
      src="http://www.sclance.com/pngs/rock-paper-scissors-png/rock_paper_scissors_png_1176896.png"
    />
  </styled.header>
</styled.div>
`;
